app/assets/stylesheets/course/forum/posts.scss

```diff
@@ -0,0 +1,5 @@
+.course-forum-posts{
+  &.reply {
+    @include forum_post;
```

Comment:
Name of mixin `forum_post` should be written in all lowercase letters with hyphens instead of underscores